components/progress/Steps.tsx

```diff
@@ -6,6 +6,7 @@ import { getSize } from './utils';
 
 interface ProgressStepsProps extends ProgressProps {
   steps: number;
+  stepRound?: 'floor' | 'ceil' | 'round';
```

Comment:
How about `rounding`?

```suggestion
  rounding?: 'floor' | 'ceil' | 'round';
```

Reply:
@afc163 yeah, sound great, it more flexible